dtfs-central-api/src/cron-scheduler-jobs/delete-record-correction-request-transient-form-data/index.test.ts

```diff
@@ -0,0 +1,79 @@
+import { LessThan } from 'typeorm';
+import { deleteRecordCorrectionRequestTransientFormData, deleteRecordCorrectionRequestTransientFormDataJob } from '.';
+import { FeeRecordCorrectionRequestTransientFormDataRepo } from '../../repositories/fee-record-correction-request-transient-form-data-repo/fee-record-correction-request-transient-form-data.repo';
+
+describe('delete-record-correction-request-transient-form-data', () => {
+  jest.mock('typeorm', () => ({
+    LessThan: jest.fn(),
+  }));
+
+  const mockDelete = jest.fn();
+
+  beforeAll(() => {
+    jest.useFakeTimers();
+  });
+
+  afterAll(() => {
+    jest.useRealTimers();
+  });
+
+  beforeEach(() => {
+    jest.resetAllMocks();
+    FeeRecordCorrectionRequestTransientFormDataRepo.delete = mockDelete;
+  });
+
+  describe('deleteRecordCorrectionRequestTransientFormData', () => {
+    beforeEach(() => {
+      jest.resetAllMocks();
+      FeeRecordCorrectionRequestTransientFormDataRepo.delete = mockDelete;
+      jest.useFakeTimers().setSystemTime(new Date('2025-01-02 12:10:00'));
+    });
+
+    it('should delete records older than one day', async () => {
+      await deleteRecordCorrectionRequestTransientFormData();
+
+      expect(mockDelete).toHaveBeenCalledTimes(1);
+
+      expect(mockDelete).toHaveBeenCalledWith({
+        lastUpdatedAt: LessThan(new Date('2025-01-01:12:10:00')),
+      });
+    });
```

Comment:
Thanks for the changes - this is much clearer with the fake timers. Can you also add a test where the system time is the first day of a month to cover that edge case please?

Reply:
done